seed/supabase-twitter-clone/README.md

```diff
@@ -285,54 +235,26 @@ const seed = await createSeedClient();
 // Reset the database, keeping the structure intact
 await seed.$resetDatabase()
 
-// Create 3 records in the HttpResponses table
-await seed.HttpResponses(x => x(3))
+// ...
 ```
 
-Now, let's edit our `seed.ts` file to generate some tweets:
-
-```ts
-await seed.$resetDatabase()
-
-// Generate 10 tweets
-await seed.tweets(x => x(10))
-```
+### Writing the Seed Script
 
-After running `npx tsx seed.ts`, we encounter an error related to invalid `avatar_url` in the Next.js images. To fix this, we adjust the `avatar_url` generation in our `seed.ts`:
+First we need to change `seed.ts` to create some users using the Supabase SDK.
 
 ```ts
-import { faker } from '@snaplet/copycat';
-
-const seed = await createSeedClient({
-  models: {
-    profiles: {
-      data: {
-        avatarUrl: ({ seed }) => faker.image.avatarGitHub(),
-      }
-    }
-  }
-});
-
-await seed.$resetDatabase()
-
-// Generate 10 tweets with valid avatar URLs
-await seed.tweets(x => x(10))
```

Comment:
Running this script failed for me. The db relies on a `users` metadata field being populated with the avatarUrl, name, user_name, so that the trigger can use these fields to create the corresponding profile: https://github.com/snaplet/examples/blob/main/seed/supabase-twitter-clone/supabase/migrations/20240312180754_remote_schema.sql#L41-L43

I tried create this metadata field, it worked but then the next error was that `@snaplet/seed` tried automatically create a profile rows when creating these tweets, so we ended up with duplicate ids (since db created some of the ids via trigger, and `@snaplet/seed` also did).

At this point though, I figured if the next thing we're doing is showing users a how to create the users+profiles using the supabase sdk, then using `connect` to connect the profiles up to the tweets, then maybe we just show them that flow, and this first example where we create the tweets without the users isn't needed.

So I changed the readme accordingly.